dev-docs/publisher-api-reference.md

```diff
@@ -549,33 +549,151 @@ Request bids. When `adUnits` or `adUnitCodes` are not specified, request bids fo
 
 ### pbjs.addAdUnits(Array)
 
-Define ad units and their corresponding header bidding bidders' tag IDs.  For usage examples, see [Getting Started]({{site.baseurl}}/dev-docs/getting-started.html).
+Takes an array of one or more ad unit objects and adds it to the Prebid auction.  For usage examples, see [Examples](#addAdUnits-Examples) below and the [Getting Started]({{site.baseurl}}/dev-docs/getting-started.html) page.
 
-**Kind**: static method of [pbjs](#module_pbjs)
++ [Ad Unit Properties](#addAdUnits-AdUnitProperties)
++ [Examples](#addAdUnits-Examples)
+
+<a name="addAdUnits-AdUnitProperties">
+
+#### Ad Unit Properties
+
+See the table below for the list of properties on the ad unit.  For example ad units, see the [Examples](#addAdUnits-Examples) below.
 
 {: .table .table-bordered .table-striped }
-| Param | Type | Description |
-| --- | --- | --- |
-| Array | `Object` &#124; `Array of objects` | of adUnits or single adUnit Object. |
+| Name         | Scope    | Type          | Description                                                                                                                                                                       |
+|--------------+----------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `code`       | Required | String        | Unique identifier that you create and assign to this ad unit.  Used to set query string targeting on the ad. If using GPT, we recommend setting this to slot element ID.          |
+| `sizes`      | Required | Array         | All the sizes that this ad unit can accept.                                                                                                                                       |
```

Comment:
this type can be of the form `array[number]` or `array[array[number]]`. Might be worth showing an example of that

Reply:
Also might add a note here that preference is to define sizes on `mediaTypes.banner.sizes`

Reply:
@mkendall07 I don't see any examples of anything using `mediaTypes.banner` in the Prebid.js repo, except in some bidder adapter code (search results pasted below).  Is there somewhere that I can see what fields are available to banners?  Or is it a per-bidder thing?

Here's my search result for `banner: ` in the prebid.js repo (sorry if long):